src/GeneratedInjectorDelegator.php

```diff
@@ -22,7 +22,7 @@ public function __invoke(ContainerInterface $container, $name, callable $callbac
     {
         $config = $container->has('config') ? $container->get('config') : [];
         $aotConfig = $config['dependencies']['auto']['aot'] ?? [];
-        $namespace = empty($aotConfig['namespace']) ? 'Laminas\Di\Generated' : $aotConfig['namespace'];
+        $namespace = $aotConfig['namespace'] ?? 'Laminas\Di\Generated';
```

Comment:
For me the test was the only reason. I'll move it into a separate PR. :+1:
Please note that the test `GeneratedInjectorDelegatorTest::testProvidedNamespaceIsNotAString()` will still fail.

Reply:
> Please note that the test `GeneratedInjectorDelegatorTest::testProvidedNamespaceIsNotAString()` will still fail.

It's fine for me to handle this in a separate Issue/PR since it's not related to the configurable logger feature and it is not an issue introduced with this PR.

@froschdesign are there any objections with handling it this way?